src/views/Drawer.tsx

```diff
@@ -243,6 +243,8 @@ export default class DrawerView extends React.PureComponent<Props> {
 
   private isStatusBarHidden: boolean = false;
 
+  private isSpringManuallyTriggered = new Value<number>(FALSE);
```

Comment:
This should probably be 
```suggestion
  private isSpringManuallyTriggered = new Value<Binary>(FALSE);
```